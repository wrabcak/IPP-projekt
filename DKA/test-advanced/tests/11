({s, q1, q2, f},
{'a'},
{
s ''-> q1,
q1'' -> q2,
q2 'a' -> f
},
s,
{f})
