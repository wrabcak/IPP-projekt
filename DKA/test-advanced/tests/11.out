(
{f, q1, q2, s},
{'a'},
{
q1 'a' -> f,
q2 'a' -> f,
s 'a' -> f
},
s,
{f}
)